modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java

```diff
@@ -580,8 +563,9 @@ public CommerceShipment updateCommerceShipment(
 		}
 
 		CommerceAddress commerceAddress = _updateCommerceShipmentAddress(
-			commerceShipment, name, description, street1, street2, street3,
-			city, zip, regionId, countryId, phoneNumber, serviceContext);
+			null, commerceShipment, name, description, street1, street2,
```

Comment:
I think here we need to do the same thing of the name (so saving in a variable the ERC and in the if `if (commerceAddress != null) ` set the ERC with the "previous one"